config/repo-values.yaml

```diff
@@ -476,3 +478,7 @@ sync:
       url: https://rasooll.github.io/helm-charts
     - name: openebs
       url: https://openebs.github.io/charts
+    - name: pulsar
+      url: https://helm.kafkaesque.io
+    - name: hkube
+      url: https://github.com/kube-HPC/helm
```

Comment:
The hkube url needs to be to a Helm repository rather than a GitHub or Git repo. Looking at your docs this would be something like https://hkube.io/helm/dev.